web/main/admin.py

```diff
@@ -107,7 +107,7 @@ def log_deletion(self, request, object, object_repr):
 
     actions = None  # use ['delete_selected'] to allow delete action
     formfield_overrides = {
-        fields.JSONField: {"widget": JSONEditorWidget},
+        JSONField: {"widget": JSONEditorWidget},
```

Comment:
This has no effect on this PR, but I noticed this was now incorrect for Django 3.2 when I first implemented one of the fields as JSON.